core-concepts/streams/stream-names.md

```diff
@@ -25,9 +25,9 @@ A _category_ stream name does not have an ID. For example, the stream name for t
 
 ## Category Types
 
-Category stream names can be broken down into sub-categories, or _types_.
```

Comment:
I believe that the new sentence is in error -- category streams names refer to category streams. It is true that category stream names can include types, but entity stream names can include them as well. `someStream:snapshot-123`, for instance, is an entity stream whose category is `someStream:snapshot`. The stream will pertain to `someStream-123`, but the subtype of `snapshot` indicates that the stream will store snapshots, not events (as would be the case for `someStream-123`). Category streams can also have types, for instance `someStream:command` is a category stream containing command messages for all entities of the category `someStream`.

Reply:
Okay I see references to the term "category" all over this section and I think that is where the confusion is coming from.  From what I've read further down and from what I understand you're telling me it seems like the notion of "types" associated with a stream is not relevant to the type of stream that it is (entity vs category streams).

Given that, would it make sense to replace references to "category types" in this section just with "types"?

> Stream names can also have an optional list of _types_.  
>
> One of the more common types associated with a category stream is the `command` type. It is used to indicate a stream that is used to transport _commands_ rather than _events_.
>
> For example, a stream name for the stream that holds commands for the `account-123` entity would be named `account:command-123`.
>
> The stream name is separated from the type list by the `:` character.
>
> Types can be present in both entity stream names as well as category stream names.
> 
> For example, the stream name for all commands for the `account` category would be `account:command`.

Reply:
That reads much better. It might be better for the first sentence to read: "Stream categories can also have an optional list of _types_."

A note about saying "stream categories" vs. "category streams" -- "Category stream" is a particular term that refers to a particular kind of stream -- the one that includes all entity streams of a given category. However, "stream categories" just refers to the categories that all streams have.

Here's a quick breakdown of how to parse a stream. Apologies if you already got this memorized, but here it is:

Given a stream name of `someStream:someType+otherType-123`

- the stream ID is `123`
- the category is `someStream:someType+otherType`
- the entity is `someStream`
- the types are `someType` and `otherType`

Given a category stream name of `someStream:someType+otherType` , I can _still_ say the category is `someStream:someType+otherType`, the entity name is `someStream`, and the types are `someType` and `otherType`.

Note that in the case of a category stream, the "entity name" effectively refers to the "class" of entities represented by the category stream. I don't think that's relevant to what you're doing, but I thought it might be a good idea to include it for completeness sake.